### ISSUE TITLE
Why do you need to capture analytics data?

### ISSUE THREAD

**Issue by lodenrogue** [wolfia-app/gpt-code-search#1]
I noticed that analytics data is being sent here: https://github.com/wolfia-app/gpt-code-search/blob/main/core/analytics.py

Why do you need that for code search to work?

**Comment by narenmanoharan** [comment]
Hey, @lodenrogue!

Added some context here: https://github.com/wolfia-app/gpt-code-search#analytics

TLDR: We don't need it for code search to work, but it's for observability and understanding how many people use the tool and how useful it is.

I find it pretty helpful to know that real users are using the tool we've built. This helps me stay motivated and invest more time continuously improving it. 

We do not log any identifiable information or any queries or responses. I've exhaustively listed all the metrics we collect in the link above. If you don't feel comfortable, feel free to turn it off and continue using the tool. It should be a one-time kill switch, and we won't collect any metrics even in future updates.
